services/api/src/dao/project.js

```diff
@@ -179,7 +179,8 @@ const addProject = ({ sqlClient }) => async (cred, input) => {
         ${input.pullrequests ? ':pullrequests' : '"true"'},
         ${input.production_environment ? ':production_environment' : 'NULL'},
         ${input.auto_idle ? ':auto_idle' : '1'},
-        ${input.storage_calc ? ':storage_calc' : '1'}
+        ${input.storage_calc ? ':storage_calc' : '1'},
+        ${input.environment_limit ? ':environment_limit' : '2'}
```

Comment:
this `2` means that the default is a limit of two environments?
I don't think that's a good default, as we probably will run into issues with that.
Let's do the default at 5  :)

Reply:
default to 5 now.